Scoop/Controller/Ride Posting/PostRideTripDetailsViewController.swift

```diff
@@ -74,11 +74,11 @@ class PostRideTripDetailsViewController: PostRideViewController {
                   }
             
             let dateFormatter = DateFormatter()
-            dateFormatter.dateFormat = "M/dd/yy @ h:mm a"
+            dateFormatter.dateFormat = "yyyy-MM-dd"
             
             NetworkManager.shared.currentRide.minTravelers = travelerCountLower
             NetworkManager.shared.currentRide.maxTravelers = travelerCountUpper
-            NetworkManager.shared.currentRide.departureDatetime = dateFormatter.string(from: date)
+            NetworkManager.shared.currentRide.departureDatetime = dateFormatter.string(from: self.datePicker.date)
```

Comment:
remove `self`

Reply:
Its in a UIAction so I think I need to call it explicitly